Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -84,7 +84,7 @@ static inline void updateQueryForGroupIfNecessary(NSMutableDictionary *dictionar
 }
 #endif
 
-static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin)
+static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin, NSString *topOrigin, bool crossOrigin)
```

Comment:
I recommend passing a `WebAuthn::Scope` for last parameter since that's what we want in the end. We don't like boolean parameters like this since it makes call sites look confusing when they pass false/true.

Also, can we add default parameter values for the last 2 parameters like somebody already suggested in an earlier review? It would avoid having to update some of the call sites.